### PR TITLE
string localization for DynamoCoreWpf

### PR DESCRIPTION
**Pull request by nguyen-binh-minh**
hi @Benglin , please review and merge

### CODE REVIEW

**Comment by lukechurch** [comment]
DNM until we've discussed pls.

**Comment by nguyen-binh-minh** [comment]
@Benglin , PTAL

File: src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs [hunk 1]
```diff
@@ -923,14 +923,14 @@ public FileDialog GetSaveDialog(WorkspaceModel workspace)
             if (workspace == HomeSpace)
             {
                 ext = ".dyn";
-                fltr = "Dynamo Workspace (*.dyn)|*.dyn";
+                fltr = Resources.FileDialogDynamoWorkspace;
```
Comment (Benglin) [inline src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs]: Good catch!

File: src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs [hunk 2]
```diff
@@ -1752,13 +1753,10 @@ private bool CanFitView(object parameter)
...
-            string fileFilter = "Library Files (*.dll, *.ds)|*.dll;*.ds|"
-                              + "Assembly Library Files (*.dll)|*.dll|"
-                              + "DesignScript Files (*.ds)|*.ds|"
-                              + "All Files (*.*)|*.*";
-
+            string[] fileFilter = {Resources.FileDialogLibraryFiles, Resources.FileDialogAssemblyFiles, 
+                                   Resources.FileDialogDesignScriptFiles, Resources.FileDialogAllFiles};
             OpenFileDialog openFileDialog = new OpenFileDialog();
-            openFileDialog.Filter = fileFilter;
+            openFileDialog.Filter = String.Join("|", fileFilter);
```
Comment (Benglin) [inline src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs]: Nice!

File: src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs [hunk 3]
```diff
@@ -851,7 +851,7 @@ private void LoadSamplesMenu()
                 {
                     var showInFolder = new MenuItem
                     {
-                        Header = "Show In Folder",
+                        Header = Wpf.Properties.Resources.DynamoViewHelpMenuShowInFolder,
```
Comment (Benglin) [inline src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs]: Why can't I find this in RESX file? Is it an existing one? The same for the following two.

**Comment by Benglin** [comment]
Hi @nguyen-binh-minh, please make sure the above strings are in place, then I can merge this in. Thanks!

File: src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs (hunk 3 above)
Comment (nguyen-binh-minh) [inline src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs]: This one is [here](https://github.com/DynamoDS/Dynamo/blob/localization/src/DynamoCoreWpf/Properties/Resources.resx#L799) .

For the other two: [1](https://github.com/nguyen-binh-minh/Dynamo/blob/tagging/src/DynamoCoreWpf/Properties/Resources.resx#L1477), [2](https://github.com/nguyen-binh-minh/Dynamo/blob/tagging/src/DynamoCoreWpf/Properties/Resources.resx#L1480)

I don't know why they don't show up in the diff.
@Benglin PTAL

**Comment by Benglin** [comment]
Thanks for the hardwork, @nguyen-binh-minh! LGTM!

**Pull request merged**